### PR TITLE
Helper Work Made Accurate

### PR DESCRIPTION
**Pull request by PotatoTomahto**
## About The Pull Request
- Starting qliphoth lowered to 2
- Good work no longer increases qliphoth
- Normal work now has 40% probability to lower qliphoth, bad work has 80% instead of guaranteed
- Added gift message

## Why It's Good For The Game
Why was reddit so easy to contain before? This makes it accurate to LC and an actual HE threat.

## Changelog
:cl:
tweak:  all-around-helper 
:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/abnormality/he/helper.dm [hunk 1]
```diff
@@ -38,6 +38,7 @@
 		/datum/ego_datum/armor/grinder
 		)
 	gift_type =  /datum/ego_gifts/grinder
+	gift_message = "Contamination scan complete. Initiating cleaning protocol."
```
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/abnormality/he/helper.dm:41]: I'd rather you not change it everywhere. The message appears when someone receives the gift, it should be as simple as "You have received the gift from X" for most abnormalities.

File: code/modules/mob/living/simple_animal/abnormality/he/helper.dm [hunk 2]
```diff
@@ -161,13 +162,14 @@
 				been_hit += L
 	addtimer(CALLBACK(src, .proc/do_dash, move_dir, (times_ran + 1)), 1)
 
-/* Work effects */
```
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/abnormality/he/helper.dm:164]: Why delete the comment?

File: code/modules/mob/living/simple_animal/abnormality/he/helper.dm [hunk 3]
```diff
@@ -161,13 +162,14 @@
...
-/mob/living/simple_animal/hostile/abnormality/helper/success_effect(mob/living/carbon/human/user, work_type, pe)
-	datum_reference.qliphoth_change(1)
+/mob/living/simple_animal/hostile/abnormality/helper/neutral_effect(mob/living/carbon/human/user, work_type, pe)
+	if(prob(40))
+		datum_reference.qliphoth_change(-1)
 	return
 
 /mob/living/simple_animal/hostile/abnormality/helper/failure_effect(mob/living/carbon/human/user, work_type, pe)
-	datum_reference.qliphoth_change(-1)
+	if(prob(80))
```
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/abnormality/he/helper.dm:171]: Why add a chance here?

**Review by Kitsunemitsu — CHANGES_REQUESTED** [review]
Update records. It is not optional.

**Review by Kitsunemitsu — APPROVED** [review]
Approved.

**Pull request merged**